samples/open-telemetry/customizing/Core_9/Sample/Program.cs

```diff
@@ -3,64 +3,55 @@
 using OpenTelemetry.Resources;
 using OpenTelemetry.Trace;
 using System;
-using System.Threading.Tasks;
 
-public static class Program
-{
-    public static async Task Main()
-    {
-        Console.Title = "CustomTelemetry";
+Console.Title = "MyEndpoint";
```

Comment:
Changed the name to make it more clear in the traces that this is an NServiceBus endpoint
![image](https://github.com/user-attachments/assets/da4bd780-b2d3-4d20-8ad7-9430ffe268de)